### PR TITLE
minimega: allow scripted multi-file VNC playback

### PR DESCRIPTION
**Pull request by floren**
In addition to the existing PointerEvent and KeyEvent in VNC playback,
files can now include a LoadFile directive. This will cause the specified
file to be loaded and played. The format follows the usual VNC playback
format, e.g.:

	0:LoadFile,browse_slashdot.vnc
	10000000000:LoadFile,/home/john/recordings/reboot_windows.vnc

The filenames may be relative to the location of the current file,
or they can be absolute. Timestamps are honored as usual.

### CODE REVIEW

File: src/minimega/vnc.go [hunk 1]
```diff
@@ -234,10 +237,42 @@ func (v *vncKBPlayback) Run() {
 			v.err = res.Write(v.Conn)
 		} else if res, err := vnc.ParsePointerEvent(s[1]); err == nil {
 			v.err = res.Write(v.Conn)
+		} else if file, err := ParseLoadFileEvent(s[1]); err == nil {
+			if !filepath.IsAbs(file) {
+				// Our file is in the same directory as the parent
+				file = filepath.Join(filepath.Dir(v.file.Name()), file)
+			}
+			log.Warn("loading file %v", file)
```
Comment (jcrussell) [inline src/minimega/vnc.go]: Probably should change to log.Debug.

**Comment by jcrussell** [comment]
Otherwise, LGTM.

**Comment by floren** [comment]
Fixes #345

**Comment by djfritz** [comment]
fixes #347

**Pull request merged**